CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 1.4.33 (2024-03-12)
+
+## Edge Agent
+### Bug fixes
+* Ensure agent and hub use logger for stack traces by @Gunni ( [4ae6c29](https://github.com/Azure/iotedge/commit/4ae6c29c771e1ce087c15ba6079c19d2f9f44b1e) )
+
+## Edge Hub
+### Bug fixes
+* Ensure agent and hub use logger for stack traces by @Gunni ( [4ae6c29](https://github.com/Azure/iotedge/commit/4ae6c29c771e1ce087c15ba6079c19d2f9f44b1e) )
+
+## aziot-edge
+### OS support
+* Add support for Snap amd64/arm64 packages ( [c38e0c8](https://github.com/Azure/iotedge/commit/c38e0c896d7b1bc7dc338b8ec0637daaeb6d27b3) )
+
+### Bug fixes
+* Fix apt purge --autoremove on Debian/Ubuntu ( [6c34f4b](https://github.com/Azure/iotedge/commit/6c34f4bdbd36de74f98ce88213ae0b11b9ffb909) )
```

Comment:
nit: `apt purge --autoremove`

Reply:
Ah, good catch. At this point I'm not going to worry about it, but I'll update the release page accordingly.